### PR TITLE
Add fbp

### PR DESCRIPTION
**Pull request by epapoutsellis**

### CODE REVIEW

**Comment by jakobsj** [comment]
This looks good. Needs a bit of tidying up though. As a first step please
- remove old files 
- remove old code lines current commented out
- move the test_xxx files to the private repo (or convert to unit tests), as these are for internal testing, not to be shipped with software.

Once done, send a message and will review again.

**Comment by epapoutsellis** [comment]
Done

File: Wrappers/Python/ccpi/astra/operators/AstraProjector3DSimple.py [hunk 1]
```diff
@@ -60,9 +69,88 @@ def domain_geometry(self):
...
+        
+        if self.sinogram_geometry.geom_type == 'cone':
+            
+            self.fdk = AstraFDK(volume_geometry = self.volume_geometry,
+                            sinogram_geometry = self.sinogram_geometry,
+                            filter_type = filter_type)  
+            self.fdk.set_input(DATA)
+            out = self.fdk.get_output()
+        
+            return out
```
Comment (jakobsj) [inline Wrappers/Python/ccpi/astra/operators/AstraProjector3DSimple.py]: I think we decided to not have FBP inside the operator, but just as the processor? Or maybe not? I see there is also an FBP in AstraProjectorSimple?

File: Wrappers/Python/ccpi/astra/operators/AstraProjector3DSimple.py [hunk 2]
```diff
@@ -60,9 +69,88 @@ def domain_geometry(self):
...
+#            raise ValueError('We need cone geometry')
+#            
+#        self.fdk = AstraFDK(volume_geometry = self.volume_geometry,
+#                            sinogram_geometry = self.sinogram_geometry,
+#                            filter_type = filter_type)  
+#        self.fdk.set_input(DATA)
+#        out = self.fdk.get_output()
+#        
+#        return out 
+
```
Comment (jakobsj) [inline Wrappers/Python/ccpi/astra/operators/AstraProjector3DSimple.py]: Delete

File: Wrappers/Python/ccpi/astra/operators/AstraProjectorSimple.py [hunk 3]
```diff
@@ -63,7 +76,30 @@ def domain_geometry(self):
...
         return self.sinogram_geometry    
     
+    def FBP(self, DATA, filter_type):
+    
+        self.fbp.filter_type = filter_type
+        self.fbp.set_input(DATA)
+        out = self.fbp.get_output()
+        
+        return out    
+
```
Comment (jakobsj) [inline Wrappers/Python/ccpi/astra/operators/AstraProjectorSimple.py]: Should this stay or be deleted to have only FBP as a processor and not as a method in the operator? Can't remember what we decided.

**Comment by epapoutsellis** [comment]
Done, remove everything from operators. Now the user is calling FBP(processor) and pass the two geometries filter

**Comment by jakobsj** [comment]
BTW, is this PR also incorporating changes from any related PRs such as #25, #32, #33?  If so, should close those PRs, once this PR has been merged.

**Pull request merged**